src/main/java/net/snowflake/ingest/streaming/internal/StreamingIngestStage.java

```diff
@@ -46,7 +47,7 @@ class StreamingIngestStage {
   private static final ObjectMapper mapper = new ObjectMapper();
   private static final long REFRESH_THRESHOLD_IN_MS =
       TimeUnit.MILLISECONDS.convert(1, TimeUnit.MINUTES);
-  static final int MAX_RETRY_COUNT = 1;
+  static final int DEFAULT_UPLOAD_RETRY_COUNT = 10;
```

Comment:
Any reason we're adding the retry count? I don't recall we're hitting any transient issue that could be solved by retrying, maybe 3 is more reasonable than 10?

Reply:
We are seeing occasional chunk rejections due to out-of-order chunks (
error code 21), where one of the reasons can be a failing upload. I will change it to 5 retries, so that we retry at least for a few seconds (if there is a network glitch).